app/views/topics/show.html.erb

```diff
@@ -1,27 +1,20 @@
 <%#
+#
 # @file
-# This shows a topic view. It is used.
-# The first commented out section used to be a card at the top showing all
-# info. Now we're moving towards most screens looking the same. The
-# consequence of accessing data from this view is that you can't remove
-# the topic that corresponds to the page you're on. Originally, accessing this
-# page showed the topic with its neighbours arrayed around. Now it shows the
-# same, but there's no cues to say which topic's page you're on. So when the
-# map recenters on a new topic, it's like you're on that topic's page.
-# Nice, but the URL and being unable to remove the root node still hamper that
-# experience.
+# This shows a topic and its siblings.
 # URL: /topics/<topicid>
 #
-#%>
+%>
 
 <% content_for :title, @topic.name + " | Metamaps" %> 
 <% content_for :mobile_title, @topic.name %> 
 <script>
-    Metamaps.currentSection = "topic";
-    Metamaps.currentPage = <%= @topic.id.to_s %>;
-    Metamaps.Active.Topic = <%= @topic.to_json(user: current_user).html_safe %>;
-    Metamaps.Creators = <%= @allcreators.to_json.html_safe %>;
-    Metamaps.Topics = <%= @alltopics.to_json(user: current_user).html_safe %>;
-    Metamaps.Synapses = <%= @allsynapses.to_json.html_safe %>;
-    Metamaps.Visualize.type = "RGraph";
+  Metamaps.currentSection = "topic"
+  Metamaps.currentPage = <%= @topic.id.to_s %>
+  Metamaps.ServerData = Metamaps.ServerData || {}
+  Metamaps.ServerData.ActiveTopic = <%= @topic(user: current_user).to_json.html_safe %>
+  Metamaps.ServerData.Creators = <%= @allcreators.to_json.html_safe %>
+  Metamaps.ServerData.Topics = <%= @alltopics(user: current_user).to_json.html_safe %>
+  Metamaps.ServerData.Synapses = <%= @allsynapses.to_json.html_safe %>
+  Metamaps.Visualize.type = "RGraph"
```

Comment:
This is different than in the case of ForceDirected where you changed it to `Metamaps.ServerData.VisualizeType`